admin/class-social-admin.php

```diff
@@ -240,48 +240,4 @@ public function og_data_compare( $post ) {
 			}
 		}
 	}
-
-
-	/********************** DEPRECATED METHODS **********************/
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.